docker/local/deployment/docker-compose.yml

```
@@ -34,6 +34,8 @@ services:
       REDIS_HOST: ${REDIS_HOST}
       REDIS_PORT: ${REDIS_PORT}
       REDIS_DB_INDEX: 2
+      REDIS_CACHE_SERVICE_HOST: ${REDIS_HOST}
+      REDIS_CACHE_SERVICE_PORT: ${REDIS_PORT}
```

```suggestion
      REDIS_CACHE_SERVICE_HOST: ${REDIS_CACHE_SERVICE_HOST}
      REDIS_CACHE_SERVICE_PORT: ${REDIS_CACHE_SERVICE_PORT}
```

Is it possible to separate both service? One is used for the queue and the other for caching functionalitys

---

Done